packages/gatsby-adapter-netlify/src/index.ts

```diff
@@ -39,6 +39,9 @@ const createNetlifyAdapter: AdapterInit<INetlifyAdapterOptions> = options => {
             `[gatsby-adapter-netlify] using @netlify/cache-utils restore`
           )
           return await utils.restore(directories)
+          reporter.info(
+            `[gatsby-adapter-netlify] Found a Gatsby cache. We're about to go FAST. ⚡`
+          )
```

Comment:
those logs are unreachable after `return`
```suggestion
          const didRestore = await utils.restore(directories)
          if (didRestore) {
            reporter.info(
              `[gatsby-adapter-netlify] Found a Gatsby cache. We're about to go FAST. ⚡`
            )
          }
          return didRestore
```

Reply:
No wonder why I thought this was such a small change. I completely missed the return statement, so sorry to have you do the work in the end 🤦🏻